jenkins/automation/vra_infra_provisioner.groovy

```diff
@@ -0,0 +1,100 @@
+pipeline {
```

Comment:
Jenkins pipeline provided in the check-list is failing - https://eos-jenkins.colo.seagate.com/job/Release_Engineering/job/re-workspace/job/gc-space/job/test-vra-infra-provisioning/

Reply:
@shailesh-vaidya, Jenkins pipeline is failing because cloud does not have enough CPU quota.

Reply:
@gauravchaudhari02 - I removed three nodes created by me for POC. This should free up CPU quota.

Reply:
@shailesh-vaidya, job is working now - https://eos-jenkins.colo.seagate.com/job/Release_Engineering/job/re-workspace/job/gc-space/job/test-vra-infra-provisioning/40/